cibuildwheel/util.py

```diff
@@ -103,5 +107,39 @@ def get_for_python_version(self, version):
             return self.base_file_path
 
 
+BuildOptions = NamedTuple("BuildOptions", [
+    ("project_dir", str),
+    ("output_dir", str),
+    ("test_command", str),
```

Comment:
Should be optional?

Reply:
yes. I think it is empty string when undefined. Or maybe it should be setted to default as empty sting (in main)?